[2023-05-02T06:02:45.816+0000] {processor.py:156} INFO - Started process (PID=33) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:02:45.836+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:02:45.841+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:02:45.840+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:02:45.875+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:02:45.870+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:02:45.878+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:02:46.063+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.265 seconds
[2023-05-02T06:03:16.259+0000] {processor.py:156} INFO - Started process (PID=90) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:03:16.261+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:03:16.262+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:03:16.262+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:03:16.278+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:03:16.274+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:03:16.280+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:03:16.344+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.106 seconds
[2023-05-02T06:03:46.888+0000] {processor.py:156} INFO - Started process (PID=149) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:03:46.928+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:03:46.931+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:03:46.931+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:03:46.982+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:03:46.978+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:03:46.983+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:03:47.132+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.250 seconds
[2023-05-02T06:04:17.469+0000] {processor.py:156} INFO - Started process (PID=207) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:04:17.470+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:04:17.471+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:04:17.471+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:04:17.477+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:04:17.475+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:04:17.479+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:04:17.867+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.402 seconds
[2023-05-02T06:04:48.200+0000] {processor.py:156} INFO - Started process (PID=264) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:04:48.203+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:04:48.207+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:04:48.206+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:04:48.222+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:04:48.218+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:04:48.223+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:04:48.256+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.067 seconds
[2023-05-02T06:05:18.641+0000] {processor.py:156} INFO - Started process (PID=322) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:05:18.642+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:05:18.644+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:05:18.644+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:05:18.650+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:05:18.648+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:05:18.651+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:05:18.678+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.041 seconds
[2023-05-02T06:05:48.987+0000] {processor.py:156} INFO - Started process (PID=380) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:05:48.989+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:05:48.992+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:05:48.992+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:05:49.000+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:05:48.998+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:05:49.001+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:05:49.026+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.051 seconds
[2023-05-02T06:06:19.551+0000] {processor.py:156} INFO - Started process (PID=438) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:06:19.553+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:06:19.556+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:06:19.555+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:06:19.565+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:06:19.563+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:06:19.566+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:06:19.625+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.081 seconds
[2023-05-02T06:06:50.052+0000] {processor.py:156} INFO - Started process (PID=496) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:06:50.055+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:06:50.057+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:06:50.057+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:06:50.066+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:06:50.064+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:06:50.067+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:06:50.094+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.057 seconds
[2023-05-02T06:07:20.342+0000] {processor.py:156} INFO - Started process (PID=554) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:07:20.343+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:07:20.345+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:07:20.344+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:07:20.349+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:07:20.348+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:07:20.350+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:07:20.373+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.036 seconds
[2023-05-02T06:07:50.784+0000] {processor.py:156} INFO - Started process (PID=612) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:07:50.786+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:07:50.787+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:07:50.787+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:07:50.792+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:07:50.791+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:07:50.793+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:07:50.815+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.034 seconds
[2023-05-02T06:08:21.147+0000] {processor.py:156} INFO - Started process (PID=670) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:08:21.149+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:08:21.151+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:08:21.151+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:08:21.157+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:08:21.155+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:08:21.157+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:08:21.184+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.044 seconds
[2023-05-02T06:08:52.073+0000] {processor.py:156} INFO - Started process (PID=728) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:08:52.074+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:08:52.076+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:08:52.075+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:08:52.083+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:08:52.081+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:08:52.083+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:08:52.107+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.040 seconds
[2023-05-02T06:09:22.838+0000] {processor.py:156} INFO - Started process (PID=786) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:09:22.840+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:09:22.843+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:09:22.843+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:09:22.855+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:09:22.853+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:09:22.856+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:09:22.884+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.057 seconds
[2023-05-02T06:09:53.866+0000] {processor.py:156} INFO - Started process (PID=844) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:09:53.867+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:09:53.869+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:09:53.869+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:09:53.875+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:09:53.873+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:09:53.876+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:09:53.899+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.042 seconds
[2023-05-02T06:10:24.237+0000] {processor.py:156} INFO - Started process (PID=902) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:10:24.239+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:10:24.241+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:10:24.241+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:10:24.250+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:10:24.247+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:10:24.251+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:10:24.286+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.056 seconds
[2023-05-02T06:10:54.340+0000] {processor.py:156} INFO - Started process (PID=960) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:10:54.341+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:10:54.342+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:10:54.342+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:10:54.347+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:10:54.346+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:10:54.348+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:10:54.372+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.036 seconds
[2023-05-02T06:11:24.936+0000] {processor.py:156} INFO - Started process (PID=1018) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:11:24.937+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:11:24.938+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:11:24.938+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:11:24.944+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:11:24.942+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:11:24.945+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:11:24.973+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.044 seconds
[2023-05-02T06:11:55.191+0000] {processor.py:156} INFO - Started process (PID=1076) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:11:55.193+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:11:55.195+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:11:55.194+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:11:55.202+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:11:55.200+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:11:55.202+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:11:55.231+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.046 seconds
[2023-05-02T06:12:25.450+0000] {processor.py:156} INFO - Started process (PID=1134) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:12:25.452+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:12:25.454+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:12:25.453+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:12:25.461+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:12:25.459+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:12:25.462+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:12:25.493+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.048 seconds
[2023-05-02T06:12:56.139+0000] {processor.py:156} INFO - Started process (PID=1192) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:12:56.141+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:12:56.143+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:12:56.142+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:12:56.151+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:12:56.149+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:12:56.152+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:12:56.184+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.051 seconds
[2023-05-02T06:13:26.719+0000] {processor.py:156} INFO - Started process (PID=1250) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:13:26.721+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:13:26.723+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:13:26.722+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:13:26.728+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:13:26.727+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:13:26.729+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:13:26.755+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.043 seconds
[2023-05-02T06:13:57.244+0000] {processor.py:156} INFO - Started process (PID=1308) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:13:57.247+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:13:57.250+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:13:57.249+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:13:57.259+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:13:57.256+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:13:57.260+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:13:57.284+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.055 seconds
[2023-05-02T06:14:27.721+0000] {processor.py:156} INFO - Started process (PID=1367) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:14:27.722+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:14:27.723+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:14:27.723+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:14:27.729+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:14:27.727+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:14:27.730+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:14:27.752+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.035 seconds
[2023-05-02T06:14:58.157+0000] {processor.py:156} INFO - Started process (PID=1425) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:14:58.158+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:14:58.159+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:14:58.158+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:14:58.164+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:14:58.163+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:14:58.165+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:14:58.187+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.034 seconds
[2023-05-02T06:15:28.236+0000] {processor.py:156} INFO - Started process (PID=1483) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:15:28.237+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:15:28.239+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:15:28.239+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:15:28.245+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:15:28.243+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:15:28.245+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:15:28.272+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.042 seconds
[2023-05-02T06:15:58.451+0000] {processor.py:156} INFO - Started process (PID=1541) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:15:58.454+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:15:58.457+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:15:58.457+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:15:58.466+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:15:58.464+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:15:58.467+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:15:58.498+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.059 seconds
[2023-05-02T06:16:28.632+0000] {processor.py:156} INFO - Started process (PID=1600) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:16:28.633+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-05-02T06:16:28.635+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:16:28.634+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:16:28.641+0000] {logging_mixin.py:149} INFO - [2023-05-02T06:16:28.639+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-05-02T06:16:28.641+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-05-02T06:16:28.665+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.038 seconds
